services/web/server/src/simcore_service_webserver/meta_function_nodes.py

```diff
@@ -0,0 +1,86 @@
+""" Nodes (services) in a project implemented with python functions (denoted meta-function nodes)
+
+
+So far, nodes in a project could be front-end, computational or dynamic. The first
+was fully implemented in the web-client (e.g. file-picker) while the two last were implemented
+independently as black-boxes (i.e. basically only i/o description known) inside of containers. Here
+we start a new type of nodes declared as "front-end" but implemented as python functions in the backend.
```

Comment:
I agree and we will do that during this sprint when we extend the different type of nodes beyond front-end/dynamic and computational. But for the moment, we need to keep compatibility.

Reply:
just call them back-end nodes no?

Reply:
it is a bit more subtle than that. We would like a classification that accounts for both where is defined and where is implemented. For instance, we will have nodes defined in the front-end but implemented in the backend (like iterators) as well as defined and implemented in the backend. 

I would really like to wait for the next PR to organize that since it also requires refactoring all the test functions to detect these different types ... and it doe snot make sense to implement here.